backend/dataall/modules/s3_datasets/services/dataset_table_service.py

```diff
@@ -70,6 +72,15 @@ def delete_table(uri: str):
             table = DatasetTableRepository.get_dataset_table_by_uri(session, uri)
             DatasetService.check_before_delete(session, table.tableUri, action=DELETE_DATASET_TABLE)
             DatasetService.execute_on_delete(session, table.tableUri, action=DELETE_DATASET_TABLE)
+
```

Comment:
Why are we not defining (lines 76-83) in DatasetService.execute_on_delete(session, table.tableUri, action=DELETE_TABLE_DATA_FILTER)?

Reply:
currently only interface registered in `DatasetService` is `S3ShareDatasetService` which was share item specific so this felt like the best place to leave in DatasetTableService since also specific only to Tables right now

Reply:
Never mind I see it know. We don't need an interface since all code is in the s3_datasets_module. If we ever make filters generic and extract them out of s3_datasets_shares then we will need to introduce the interface. Resolving for the moment